java-dynamic-sqs-listener-api/src/main/java/com/jashmore/sqs/argument/ArgumentResolutionException.java

```diff
@@ -0,0 +1,23 @@
+package com.jashmore.sqs.argument;
+
+/**
+ * Exception thrown when there was an error resolving an argument for the method.
+ *
+ * <p>This can occur if a payload is trying to be resolved but the message does not conform and cannot be correctly created. For example,
+ * you have a Java Bean being built from the message body but the message is not in the correct format.
+ */
+public class ArgumentResolutionException extends RuntimeException {
+    private static final long serialVersionUID = 1L;
```

Comment:
So what is your thinking about managing `serialVersionUID` manually. Where will you be serializing these instances? I expect it all to be JSON via Jackson in the upper layer.

Reply:
> So what is your thinking about managing serialVersionUID manually

Nope, FindBugs error so I just put it in to make it happy. I could look at removing that constraint on findbugs or ignore these exception classes.

Reply:
Yeh I would remove it from code and do what you have to do to quiet FindBugs

Reply:
Ah nevermind, no this is caused by me making compiler warnings as errors and is therefore being thrown by the maven-compiler-plugin. I am going to keep it as there is no harm.

Reply:
There are slight implications about it and it raised my eyebrows to talk enough about it. I would peek around in some other frameworks codes and see if you see it being used. I would remove it. My 2c.